testing/apmsoak/main.tf

```diff
@@ -0,0 +1,32 @@
+terraform {
+  required_version = ">= 1.1.8, < 2.0.0"
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = ">=4.27.0"
+    }
+  }
+  backend "gcs" {
+    bucket = "apmsoak-tf"
+    prefix = "/tfstate"
+  }
```

Comment:
I haven't tested using different workspaces with the GCS state set. Did you check it works well?

Reply:
Yes, tested with 2 workspaces as well as with a service account